### ISSUE TITLE
need more examples and detailed documentation

### ISSUE THREAD

**Issue by oak93** [theKashey/use-sidecar#10]
Hi,

I really loved the idea of the library. 

It has a different approach to code-splitting but I think, it is kinda hard to understand implementation even if I have read the readme file couple of times.

Tried to analyze the other libraries of yours which implement use-sidecar but did not help. 
Maybe I am the only one having some problems but is it possible to add some dummy web applications that implement the library or **some short tutorial video somewhere would be great**

To sum up, if you can add some basic examples and more documentation that would really help 🙌

Thanks @theKashey

**Comment by theKashey** [comment]
Yeah. This library is tiny, but not that easy to comprehend.

**Comment by theKashey** [comment]
I promised a sound explanation to @jherr quite a while ago, so yes - I will do that. 
When? Well, there are long weeksends
